### DISCUSSION TITLE
Is there any special config for safari?

### DISCUSSION THREAD

**Discussion by raisonjoseph** [videojs/video.js#7355] [Q&A]
I have been trying to use video.js in iOS. When start playing the iOS will take over the playback in full screen mode and shows overly over the video element. 

When I add playsinline attribute to video element, the palyback will not take over by the iOS but fullscreen toggle does not work.
Please help me

**Comment by gkatsev** [comment]
`playsinline` is what you want if you want it to not go fullscreen on iPhones. Fullscreen button should continue to work. I just tested locally and seems to be working for me. Have a live minimal test page we can take a look at?
